src/dice/ffi/metall.h

```diff
@@ -5,11 +5,13 @@
 #include <stddef.h>
 
 #include <metall/logger_interface.h>
```

Comment:
This is to make sure that the logger_interface is declared strictly before the metall c interface, right?

Reply:
No, its basically just so that its available for code completition if you want to implement it

Reply:
OK, fine for me.